lib/AST/Decl.cpp

```diff
@@ -8883,10 +8901,10 @@ bool ConstructorDecl::isObjCZeroParameterWithLongSelector() const {
   return params->get(0)->getInterfaceType()->isVoid();
 }
 
-DestructorDecl::DestructorDecl(SourceLoc DestructorLoc, DeclContext *Parent)
+DestructorDecl::DestructorDecl(SourceLoc DestructorLoc, bool Async, SourceLoc AsyncLoc, DeclContext *Parent)
```

Comment:
Nit: Run `git-clang-format` over the commits in the PR

Reply:
Done